core/controllers/reader.py

```diff
@@ -608,18 +610,44 @@ def post(self, exploration_id):
 class SolutionHitEventHandler(base.BaseHandler):
     """Tracks a learner clicking on the 'View Solution' button."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': editor.SCHEMA_FOR_EXPLORATION_ID
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'exploration_version': {
+                'schema': editor.SCHEMA_FOR_VERSION
+            },
+            'state_name': {
+                'schema': {
+                    'type': schema_utils.SCHEMA_TYPE_BASESTRING
```

Comment:
I think we usually do not use constants for these simple strings (we do not use them in schemas, in other parts of the codebase we use them).

Reply:
done.

Reply:
I have replaced those constants. Please tell me If it needs any other changes.